CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 3.0.2
+
+### Bugfix
+
+* Don't replace tags are not related to hidden custom tags regexp formats. A good example are tabs, so we might not replace tabs in the content.
```

Comment:
Fred can you try rewriting this? It doesn't make much sense to me. "We might not replace tabs" ?

`Fix replacing tags related to hidden custom tags regexp formats. E.g. tab characters`

Reply:
Docx can have tags inside <w> that's not <t> for example <tab>. If that's the case we should skip to replace this content.